tests/unit/conftest.py

```diff
@@ -166,18 +162,18 @@ def register_command_handler(
 @pytest.fixture(scope="function", name="container")
 def container_fixture(
     harness: Harness,
-    admin_credentials: Credentials,
+    admin_credentials: jenkins.Credentials,
     monkeypatch: pytest.MonkeyPatch,
     proxy_config: state.ProxyConfig,
 ) -> Container:
     """Harness Jenkins workload container that acts as a Jenkins container."""
     harness.set_can_connect("jenkins", True)
     container: Container = harness.model.unit.get_container("jenkins")
     container.push(
```

Comment:
I think there is a way to interact directly with the file system now in ops? So container push shouldn't be needed any more?

Reply:
I've pushed the change, it seems a lot more verbose.

Reply:
The benefit is that how pathlib works is common knowledge so the code is easier to understand for the developer that is less familiar with container push etc